app/components/ui/search/index.js

```diff
@@ -158,6 +165,10 @@ const Search = React.createClass( {
 	},
 
 	render() {
+		if ( true ) {
```

Comment:
Yeah, I was actually trying to get the linter to love me - but after I fixed the initial errors, he just threw at me some CSS ones... So I gave up.